parsers/discord.py

```diff
@@ -0,0 +1,154 @@
+import functools
+
+from PyQt5 import QtCore
+from PyQt5.QtWebEngineWidgets import QWebEngineView
+from PyQt5.QtWidgets import QScrollArea, QPushButton, QDialog, QGridLayout
+
+from helpers import ParserWindow, config
+
+
+JS_ADD_CSS_TEMPLATE = """(
+function() {
+    css = document.getElementById('%(name)s');
+    if (css == null) {
+        css = document.createElement('style');
+        css.type='text/css'; css.id='%(name)s';
+        document.head.appendChild(css);
+    }
+    css.innerText='%(new_css)s';
+})()"""
+CSS_SMALLER_AVATARS = """
```

Comment:
TODO: Remove this also, didn't end up using it. Opted to just let the user's config settings stand.